frontends/web/src/components/forms/select.tsx

```diff
@@ -27,16 +27,18 @@ type TSelectProps = {
     id: string;
     label?: string;
     options: TOption[];
+    extraLeftSpace?: boolean;
 } & JSX.IntrinsicElements['select']
```

Comment:
Sure. I thought this would be okay since we might have to make similar stuff (select with icon) in the future, and we can nicely pass `extraLeftSpace`. But I'm ok with your suggestion. 🙂

Reply:
And having the style inside `exchange.module.css` will force us to use `!important` as styles from `exchange` (the extra padding) will be overridden by the style from `select.module.css` : `padding: 0 var(--space-half);`. We unfortunately can't stop this from happening as CSS modules can't refer another module's class. 

So for now, I'll remove `extraLeftSpace` and use `!important` in `exchange.module.css`. Lmk what you think

Reply:
ok I see, but then the prop of the select should accept an optional icon and apply the different style depending on if there is an optional icon.